torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2087,7 +2087,7 @@ def large_1d_unique(dtype, device):
     samples.append(SampleInput(scalar, args=(0,)))
     samples.append(SampleInput(scalar, args=(0, True)))
     # no CUDA support for stable sort yet
-    if not device.startswith('cuda'):
+    if torch.device(device).type != 'cuda':
```

Comment:
Nice fix

Reply:
Even better fix would be removing this altogether because stable sort is supported on cuda.